src/sentry/templates/sentry/partial/preload-data.html

```diff
@@ -47,18 +47,31 @@
       return host + '/api/0/organizations/' + slug + suffix;
     }
 
-    var preloadPromises = {orgSlug: slug};
+    var preloadPromises = { orgSlug: slug };
```

Comment:
is prettier reformatting this 😬

Reply:
Yes 😶

Reply:
something weird going on here, wouldn't expect this